.github/workflows/ci-pre-commit.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     name: pre-commit hooks
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-      - uses: pre-commit/action@v2.0.0
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
```

Comment:
I think that the reason for

```/home/runner/.cache/pre-commit/repobd6_h4eo/py_env-python3.10/lib/python3.10/site-packages/numpy/init.pyi:636:
error: Positional-only parameters are only supported in Python 3.8 and greater [syntax]
```
is that recently the default version of `actions/setup-python` was bumped up to 3.10.
I could not in any way reproduce the issue with a local Python 3.10.